sys/include/net/gnrc/ipv6/netif.h

```diff
@@ -114,6 +114,13 @@ extern "C" {
 /**
  * @{
  * @name Flags for the interfaces
+ *
+ * @note The most-significant byte of the these flags is identical to the flags
+ *       in IPv6 router advertisements. See <a
+ *       href="https://tools.ietf.org/html/rfc4861#section-4.2">RFC4861,
+ *        section 4.2</a> and <a
+ *       href="https://tools.ietf.org/html/rfc6275#section-7.1">RFC6275,
```

Comment:
Yes, but in RFC 4861 only the `M` and `O` bit are mentioned, while the older RFCs additionally define `Prf` and `P`.